core/templates/core/irap_request/steps/cyber_sso.html

```diff
@@ -2,13 +2,20 @@
 
 {% load gds %}
 
-{% block page_title %}My page title{% endblock %}
-{% block page_heading %}My page heading{% endblock %}
+{% block page_title %}Cyber SSO{% endblock %}
+{% block page_heading %}Can SSO be used?{% endblock %}
 
 {% block step_content %}
 <form method="post" novalidate>
     {% csrf_token %}
     {% include 'gds/error-summary.html' %}
+    <h2 class="govuk-heading-m">Select all that apply</h2>
+
+    <h2 class="govuk-heading-m">Can Single-Sign on (SSO) be used?</h2>
+    {% gds_field form.can_use_sso template='gds/radio.html' conditionals=True hide_label=True divider='or' %}
+
+    <h2 class="govuk-heading-m">Can Multi-factor authentication be enabled for access?</h2>
+    {% gds_field form.can_enable_multi_factor_auth template='gds/radio.html' conditionals=True hide_label=True divider='or' %}
```

Comment:
The below should be basically the same as these fields have labels and the `gds_field` template tag will pick the correct template for us.
```suggestion
    {% gds_field form.can_use_sso %}
    {% gds_field form.can_enable_multi_factor_auth %}
```